release.config.js

```diff
@@ -2,7 +2,11 @@ module.exports = {
   repositoryUrl: 'https://github.com/PolymeshAssociation/polymesh-sdk.git',
   branches: [
     'master',
-    '22-patch',
+    {
+      name: 'patch',
+      prerelease: true,
```

Comment:
I don't think we need this as pre-release if the range works like I think it does